### PR TITLE
linting and readme update

### PR DESCRIPTION
**Pull request by matthewoconnor-development**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["seneca"]
+}
```
Comment (ckiss) [inline .eslintrc]: add newline at EOF

File: lib/sqlite-store.js [hunk 2]
```diff
@@ -1,433 +1,443 @@
...
 
-      if(!conf) {
-        conf = {};
-        //sqlite:///path/to/database;
-        var urlM = /^sqlite:\/\/(.*?)$/.exec(spec);
-        conf.database   = urlM[1];
+      if (!conf) {
+        conf = {}
+        // sqlite:///path/to/database;
+        var urlM = /^Sqlite:\/\/(.*?)$/.exec(spec)
```
Comment (ckiss) [inline lib/sqlite-store.js]: I don't think you need to change to capitalized here. it's an url format. See commented example

File: lib/sqlite-store.js [hunk 3]
```diff
@@ -1,433 +1,443 @@
...
+
+    if (!_.isEmpty(w)) {
+      for (var param in w) {
+        var fieldPlaceholder = '$' + param
+        params.push(param + ' = ' + fieldPlaceholder)
+        values[fieldPlaceholder] = w[param]
       }
 
-      wherestr = " WHERE " + params.join(' AND ');
+      wherestr = 'WHERE' + params.join('AND')
```
Comment (ckiss) [inline lib/sqlite-store.js]: this builds a native query string so removing spaces before and after keywords is not ok. you will end up having SELECT*FROMtableWHEREa==bANDc==d

File: npm-debug.log [hunk 4]
```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok
```
Comment (ckiss) [inline npm-debug.log]: remove this file from commit. you should add .gitignore and add this file there

File: package.json [hunk 5]
```diff
@@ -19,21 +20,25 @@
...
   "contributors": [
     "Peter Elger <elger.peter@gmail.com> (http://peterelger.com/)",
-    "Vito Tardia (http://vito.tardia.me)"
+    "Vito Tardia (http://vito.tardia.me)",
+    "Matthew O'Connor (https://github.com/matthewoconnor-development)"
   ],
-  "license": "BSD",
-
+  "license": "MIT",
+  "homepage": "https://github.com/senecajs/seneca-sqlite-store",
```
Comment (ckiss) [inline package.json]: add "bugs" attr too

File: package.json [hunk 6]
```diff
@@ -19,21 +20,25 @@
...
-    "lodash": "^3.10.1"
+    "lodash": "^4.3.0"
   },
 
   "devDependencies": {
-    "lab": "^5.16.0",
-    "seneca": "^0.6.5",
-    "seneca-store-test": "^0.2.2"
+    "lab": "^8.2.0",
+    "seneca": "^1.1.0",
```
Comment (ckiss) [inline package.json]: use "plugin" instead of specific version

File: test/sqlite.test.js [hunk 7]
```diff
@@ -1,41 +1,38 @@
-/*jslint node: true */
-/*global describe:true, it:true */
+/* jslint node: true */
```
Comment (ckiss) [inline test/sqlite.test.js]: remove this jslint and copyright lines

**Comment by ckiss** [comment]
remove this from commit test/db/senecatest.db and also add it to .gitignore

**Comment by ckiss** [comment]
@mihaidma can you please have a look and merge if it seems ok to you too?

**Comment by mihaidma** [comment]
looks ok, I will merge

**Pull request merged**